terraform/testdata/plan-module-variable-from-splat/mod/main.tf

```diff
@@ -1,12 +1,12 @@
 variable "param" {
-  type = "list"
+  type = list(string)
 }
 
 resource "aws_instance" "test" {
-  count    = "2"
+  count = "2"
   thing = "doesnt"
 }
 
 output "out_from_splat" {
-  value = ["${aws_instance.test.*.thing}"]
+  value = aws_instance.test.*.thing
```

Comment:
Users who unwrap the `"${}"` will get a confusing error here until they remove the `[]` (which makes sense) ... This adjustment was not removed by `fmt` automagically, but a manual update, but thought to mention it.